node/src/Pos/Block/Slog/Logic.hs

```diff
@@ -328,9 +326,3 @@ slogCommon
     -> m ()
 slogCommon newLastSlots = do
     slogPutLastSlots newLastSlots
```

Comment:
What happened to this function? Does it require no further changes?

Reply:
This logic was moved into `Pos.Update.Logic.Global`.